dev/tests/integration/testsuite/Magento/Checkout/_files/disable_all_active_shipping_methods.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
Please, reuse https://github.com/magento/graphql-ce/blob/2.3-develop/dev/tests/integration/testsuite/Magento/OfflineShipping/_files/enable_offline_shipping_methods.php
fixture

Reply:
**done**, sir

@naydav , could you please verify? 

Thank you!